app/javascript/components/lease_ups/LeaseUpsTable.js

```diff
@@ -9,14 +9,17 @@ import appPaths from '~/utils/appPaths'
 
 import { LEASE_UP_STATUS_OPTIONS, PAGE_SIZE, getLeaseUpStatusStyle } from './leaseUpsHelpers'
 
-const LeaseUpStatusCell = ({ cell, onChange, applicationId }) => {
+const LeaseUpStatusCell = ({ cell, onChange }) => {
+  const  applicationPreferenceId = cell.original.id
```

Comment:
@fddayan reviewing this more for understanding. can you provide more context on `applicationPreferenceId` usage here? why do we use it at all when updating a lease up status?